eth/downloader/statesync.go

```diff
@@ -210,6 +207,42 @@ func (d *Downloader) runStateSync(s *stateSync) *stateSync {
 	}
 }
 
+// spindownStateSync 'drains' the outstanding requests; some will be delivered and other
+// will time out. This is to ensure that when the next stateSync starts working, all peers
+// are marked as idle and de facto _are_ idle.
+func (d *Downloader) spindownStateSync(active map[string]*stateReq, timeout chan *stateReq, peerDrop chan *peerConnection, next *stateSync) *stateSync {
+	// We're exiting now, but we need to time out (or handle outstanding requests first)
+	for len(active) > 0 {
+		log.Debug("Statesync exiting...", "active peers", len(active))
```

Comment:
Is it fine to add the blank in the log?

Reply:
Oh you mean the space? Yes that's not a problem (it'll be embedded in quotation-marks)